tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -854,6 +854,9 @@ def _impl(ctx):
                         iterate_over = "user_link_flags",
                         expand_if_available = "user_link_flags",
                     ),
+                    flag_group(
```

Comment:
flag_group must have one of flag_groups and flags nonempty, which will make Bazel complain in case ctx.attr.link_libs is empty.

Reply:
Interview question :) Why was @hlopko never supposed to be hired? https://source.bazel.build/bazel/+/fa654b4878974e591d7e0bea3fa233c6b120395b:tools/cpp/lib_cc_configure.bzl;l=47

Reply:
Ha, I'm not stupid, maybe. Ignore this.

Reply:
Done.